### PR TITLE
Prisma cloud aws ec2 remediation fix

### PR DESCRIPTION
**Pull request by tomer-pan**
## Status
- [ ] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-8662

## Description
Remove deprecated policies from "Prisma Cloud Remediation - AWS EC2 Instance Misconfiguration v2" playbook and related sub-playbook.

### CODE REVIEW

**Review by efelmandar — APPROVED** [review]

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_AWS_EC2_Instance_Misconfig_v2.yml [hunk 1]
```diff
@@ -2,30 +2,11 @@ id: Prisma Cloud Remediation - AWS EC2 Instance Misconfiguration v2
...
-  - AWS Security Groups allow internet traffic from internet to CIFS port (445)
-  - AWS Security Groups allow internet traffic from internet to SQLServer port (1433)
-  - AWS Security Groups allow internet traffic from internet to SQLServer port (1434)
-  - AWS Security Groups allow internet traffic from internet to MYSQL port (3306)
-  - AWS Security Groups allow internet traffic from internet to RDP port (3389)
-  - AWS Security Groups allow internet traffic from internet to MSQL port (4333)
-  - AWS Security Groups allow internet traffic from internet to PostgreSQL port (5432)
-  - AWS Security Groups allow internet traffic from internet to VNC Listener port (5500)
-  - AWS Security Groups allow internet traffic from internet to VNC Server port (5900)
+  - AWS Default Security Group Does Not Restrict All Traffic (policy id: 2378dbf4-b104-4bda-9b05-7417affbba3f)
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloud/Playbooks/playbook-PCR_-_AWS_EC2_Instance_Misconfig_v2.yml:7]: does capitalization matter?  Should this be: AWS Default Security Group does not restrict all traffic.

File: Packs/PrismaCloud/ReleaseNotes/4_2_9.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+#### Playbooks
+
+##### Prisma Cloud Remediation - AWS EC2 Security Group Misconfiguration
+
+Removed sections related to Prisma Cloud's deprecated policies.
+##### Prisma Cloud Remediation - AWS EC2 Instance Misconfiguration v2
+
+Removed Prisma Cloud's deprecated policy IDs from task #17.
```
Comment (ShirleyDenkberg) [inline Packs/PrismaCloud/ReleaseNotes/4_2_9.md:9]: Is this a link?  Why is it underlined?

**Comment by ShirleyDenkberg** [comment]
@efelmandar Doc review completed.

File: Packs/PrismaCloud/Playbooks/playbook-PCR_-_AWS_EC2_Instance_Misconfig_v2.yml (hunk 1 above)
Comment (tomer-pan) [reply Packs/PrismaCloud/Playbooks/playbook-PCR_-_AWS_EC2_Instance_Misconfig_v2.yml:7]: The names are identical to how the appear on Prisma Cloud's side.

File: Packs/PrismaCloud/ReleaseNotes/4_2_9.md (hunk 2 above)
Comment (tomer-pan) [reply Packs/PrismaCloud/ReleaseNotes/4_2_9.md:9]: It's not a link - probably a github parsing thing.

**Pull request merged**